scripts/set-alias-page.py

```diff
@@ -151,45 +159,57 @@ def set_alias_page(file, command):
     orig_command = get_alias_page(file)
     if orig_command == command:
         return ""
-    elif orig_command == "":
-        status = "\x1b[36mpage added"
+
+    if orig_command == "":
+        status_prefix = "\x1b[36m"
+        action = "added"
     else:
-        status = "\x1b[34mpage updated"
+        status_prefix = "\x1b[34m"
+        action = "updated"
 
-    alias_name = os.path.basename(file[:-3])
-    text = (
-        templates[locale].replace("example", alias_name, 1).replace("example", command)
-    )
-    os.makedirs(os.path.dirname(file), exist_ok=True)
-    with open(file, "w") as f:
-        f.write(text)
+    if dry_run:
+        status = f"page will be {action}"
+    else:
+        status = f"page {action}"
+
+    status = f"{status_prefix}{status}\x1b[0m"
+
+    if not dry_run:  # Only write to the file during a non-dry-run
+        alias_name = os.path.basename(file[:-3])
```

Comment:
I found `file[:-3]` is used to strip `.md` extension. I think this expression can be modified for readability, for example:
```py
import os
file = "pages/linux/mkfs.ext4.md"
alias_name, _ = os.path.splitext(os.path.basename(file)) # relpace file[:-3] with splitext
print(alias_name)
```
Anyway, it depends on whether you guys think file[:-3] is fine.